sample-apps/communication/index.html

```diff
@@ -12,7 +12,11 @@
       <p id="dest-reached" style="background-color: darkgreen; display: none">
         You've reached your destination!
       </p>
+      <p id="nearest-poi-found" style="background-color: darkgreen; display: none">
+        Nearest POI is: <p id="nearest-poi-name"></p>
```

Comment:
https://www.sitepoint.com/community/t/p-inside-p/8402/2

Reply:
Changed the inner one to div